### PR TITLE
Added Device Metadata

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Networking/Models/v2/Metadata.swift [hunk 1]
```diff
@@ -86,8 +107,33 @@ public enum DocumentImageOriginValue: String, Codable {
...
+
+    var modelName: String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        let machineMirror = Mirror(reflecting: systemInfo.machine)
+        let identifier = machineMirror.children.reduce("") { identifier, element in
+            guard let value = element.value as? Int8, value != 0 else { return identifier }
+            return identifier + String(UnicodeScalar(UInt8(value)))
+        }
+        return identifier
```
Comment (tobitech) [inline Sources/SmileID/Classes/Networking/Models/v2/Metadata.swift:129]: I don't know if you have seen the format returned by this yet but I'm thinking we can return a more friendly model name that's easy to understand: e.g. `iPhone 14 Pro` instead of `iPhone15,2`

File: Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycScreen.swift [hunk 2]
```diff
@@ -53,7 +54,7 @@ struct OrchestratedBiometricKycScreen: View {
                 extraPartnerParams: extraPartnerParams,
                 skipApiSubmission: true,
                 onResult: viewModel
-            )
+            ).environmentObject(localMetadata)
```
Comment (tobitech) [inline Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycScreen.swift:57]: I see this injected in several places, but I couldn't see there the injected object was used in the views.

**Review by tobitech — CHANGES_REQUESTED** [review]

**Comment by jumaallan** [comment]
Closed - Review #233
